runtime/src/storage/mod.rs

```diff
@@ -1,8 +1,13 @@
 //! Implementations of the `Storage` trait that can be used by the runtime.
 pub mod audited;
+#[cfg(all(feature = "iouring", target_os = "linux"))]
```

Comment:
Again, recommend just using feature?

I suppose this basically implies "no-op" if not on linux when the feature is enabled?